### DISCUSSION TITLE
Bard missing image button

### DISCUSSION THREAD

**Discussion by CorvS** [statamic/cms#10309] [Troubleshooting] [answered]
Hey there!
When I create a brand new Statamic project (5.7.3) with `statamic new foobar` and then add a **Bard** field to the default **Page** blueprint with the image button enabled, it does not show up when editing the page.

Blueprint:
![blueprint](https://github.com/statamic/cms/assets/16050553/afb611d5-6f47-4d4c-b18c-0ae9abefeb0f)

Page:
![bard](https://github.com/statamic/cms/assets/16050553/42a208af-1d99-44f1-8649-89c4e9a42fde)

Did I miss anything?

**Comment by marcorieser** [comment, marked as answer]
Have you configured an assets container?

**Reply by CorvS** [reply]
I guess I was a bit hasty there. That did the trick. Thanks a lot.
